test-utils/state-viewer/src/commands.rs

```diff
@@ -490,6 +491,45 @@ pub(crate) fn check_block_chunk_existence(store: Arc<Store>, near_config: NearCo
     }
     println!("Block check succeed");
```

Comment:
Although, this is not part of this PR. It may be  a good idea to switch to using tracing, rather than !println!